docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -1826,12 +1826,14 @@
     "\n",
     "1. Use the `recover_configurations()` method to obtain a refined bitstring matrix and probability array based on the average orbital occupancy.\n",
     "1. Use the `postselect_and_subsample()` to collect batches of subsamples to diagonalize over.\n",
-    "1. Then use the batches of subsamples as arguments to the `solve_fermion()` method to obtain an approximation of the ground state."
+    "1. Then use the batches of subsamples as arguments to the `solve_fermion()` method to obtain an approximation of the ground state.\n",
```

Comment:
A little pedantic, but I guess we could say "function" instead of "method" to distinguish it from a class method.